packages/core/src/router.ts

```diff
@@ -39,3 +39,29 @@ export function newRouterExecuteTransactionRequest(
 
   return { to: getContractAddress(chainId, 'Router'), data, value };
 }
+
+export interface NewRouterExecuteWithSignerFeeTransactionOptions {
```

Comment:
Does the name need to include the "New"? Would `RouterExecuteWithSignerFeeTransactionOptions` be enough?

Reply:
My experience is function name + options, and I won't spend too much time thinking about what to omit.

Reply:
I see. Thanks for the explanation.